### PR TITLE
fix: feature flag name for vm updates

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

Made a little typo with the feature flag 🤦🏼 

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

### CODE REVIEW

**Review by dvail — APPROVED** [review]
I was very confused by this until I looked closer 😄

**Review by vjwilson — APPROVED** [review]
I did not notice before, and it really would have been OK to leave.

File: ui/apps/platform/src/types/featureFlag.ts [hunk 1]
```diff
@@ -6,4 +6,4 @@ export type FeatureFlagEnvVar =
     | 'ROX_SECURITY_METRICS_PHASE_ONE'
     | 'ROX_SYSTEM_HEALTH_PF'
     | 'ROX_VERIFY_IMAGE_SIGNATURE'
-    | 'ROX_FRONTEND_VM_UDPATES';
+    | 'ROX_FRONTEND_VM_UPDATES';
```
Comment (pedrottimark) [inline ui/apps/platform/src/types/featureFlag.ts:9]: Unlike list.go file in which pattern is to add at the end, we might form habit to add in alpha order to somewhat reduce the chance of merge conflicts when velocity increases (especially to reduce friction when backend engineers need to add feature flags).

**Pull request closed without merging**